index.html

```diff
@@ -7958,7 +7969,7 @@ <h3 id="playback-AudioBufferSourceNode">
 
   // Determine loop endpoints as applicable
   let actualLoopStart, actualLoopEnd;
-  if (loop) {
+  if (loop &amp;& buffer != null) {
```

Comment:
Isn't it just `&amp;` to get `&` instead of `&amp;&`? If so, I guess line 7373 below needs to be fixed too.

Reply:
True, you will need `&amp;&amp;` to get `&&`.

Reply:
I used `&amp;&amp;` (give me some credit :-) but html5tidy "corrects" this to `&amp;&`. So checks fail if the correct construct is used.